BUILD.md

```diff
@@ -142,6 +142,23 @@ bin/linkerd -n emojivoto stat deployments
 bin/linkerd -n emojivoto tap deploy voting
 ```
 
+#### Deploying Control Plane components with Tracing
+
+Control Plane components have the `trace-collector` flag used to enable [Distributed Tracing](https://opentracing.io/docs/overview/what-is-tracing/) for development purposes. It can be enabled globally i.e Control plane components and their proxies by using the `--control-plane-tracing` installation flag.
+
+This will configure all the components to send the traces at `linkerd-collector.linkerd-tracing:55678`
```

Comment:
I think this should be `linkerd-collector.linkerd:55678`

Reply:
Updated that to `linkerd-collector.{{.Namespace}}.svc.{{.ClusterDomain}}:55678`